tests/Unit/Domain/Test_SizeOfElement.cpp

```diff
@@ -33,6 +33,54 @@
 #include "Utilities/TMPL.hpp"
 
 namespace {
+void test_1d() noexcept {
+  INFO("1d");
+  auto map = domain::make_coordinate_map_base<Frame::Logical, Frame::Inertial>(
+      domain::CoordinateMaps::Affine(-1.0, 1.0, 0.3, 1.2));
+  const ElementId<1> element_id(0,
+                                std::array<SegmentId, 1>({{SegmentId(2, 3)}}));
```

Comment:
you can do this if you want:
```suggestion
  const ElementId<1> element_id{0, {{{2, 3}}}};
```